pkg/subsystem/lists/linux.go

```diff
@@ -0,0 +1,4757 @@
+}
+
+// Subsystem info.
+
+var shacyfmac = &Subsystem{
+	Name:        "SHA-cyfmac",
+	Lists:       []string{"SHA-cyfmac-dev-list@infineon.com"},
+	Maintainers: []string{"aspriel@gmail.com", "franky.lin@broadcom.com", "hante.meuleman@broadcom.com"},
+	Parents:     []*Subsystem{wireless},
+	PathRules: []PathRule{
+		{IncludeRegexp: "^drivers/net/wireless/broadcom/brcm80211(?:/.*)?$"},
+	},
+}
+
+var abi = &Subsystem{
+	Name:        "abi",
+	Lists:       []string{"linux-abi-devel@lists.sourceforge.net"},
+	Maintainers: []string{"hch@infradead.org"},
+	Parents:     []*Subsystem{kernel},
+	PathRules: []PathRule{
+		{IncludeRegexp: "^include/linux/personality\\.h$|^include/uapi/linux/personality\\.h$"},
+	},
+}
+
+var ac100 = &Subsystem{
+	Name:        "ac100",
+	Lists:       []string{"ac100@lists.launchpad.net"},
+	Maintainers: []string{"marvin24@gmx.de"},
+	Parents:     []*Subsystem{staging, tegra},
+	PathRules: []PathRule{
+		{IncludeRegexp: "^drivers/staging/nvec(?:/.*)?$"},
+	},
+}
+
+var accelerators = &Subsystem{
+	Name:        "accelerators",
+	Lists:       []string{"linux-accelerators@lists.ozlabs.org"},
+	Maintainers: []string{"wangzhou1@hisilicon.com", "zhangfei.gao@linaro.org"},
+	Parents:     []*Subsystem{kernel},
+	PathRules: []PathRule{
+		{IncludeRegexp: "^drivers/misc/uacce(?:/.*)?$|^include/linux/uacce\\.h$|^include/uapi/misc/uacce(?:/.*)?$"},
+	},
+}
+
+var accessrunner = &Subsystem{
+	Name:    "accessrunner",
+	Lists:   []string{"accessrunner-general@lists.sourceforge.net"},
+	Parents: []*Subsystem{usb},
+	PathRules: []PathRule{
+		{IncludeRegexp: "^drivers/usb/atm/cxacru\\.c$"},
```

Comment:
This is just 1 file, and the next acenic subsystem is the same.
Thinking if we should just drop these subsystems (merge into the parent)...

Reply:
I've added a change that removes subsystems with less than 2 matching files. Actually, that removed quite a few.